sdk/iot/doc/how_to_iot_hub_samples_windows.md

```diff
@@ -0,0 +1,408 @@
+- Make it the default startup project (right-click on `paho_iot_hub_c2d_example` project, then click on `Set as StartUp Project`);
+- Build and run the project (`F5` on most installations).
+
+On the Azure Portal,
+- Go to your Azure IoT hub page.
+- Click on "IoT devices" under "Explorers".
+- From the list of devices, click on your device (created on step 6).
+- Click on "Message to Device".
+- On "Message Body", type "Hello world!" (too cheesy? how about "Lorem Ipsum"?)
+- Click on "Send Message".
+
+Back to the Visual Studio Command prompt, verify that the message has been received by the sample:
+
+```shell
+X509 Certificate PEM Store File = C:\azure-sdk-for-c\sdk\iot\hub\samples\src\device_cert_store.pem
+X509 Trusted PEM Store File = C:\azure-sdk-for-c\sdk\iot\hub\samples\src\BaltimoreCyberTrustRoot.crt.pem
+Device ID = paho-sample-device1
+IoT Hub Hostname = myiothub.azure-devices.net
+Posting connect semaphore for client paho-sample-device1 rc 0Subscribed to topics.
+Waiting for activity. [Press ENTER to abort]
+C2D Message arrived
+Hello world!
+
+``` 
+
+Note: the sample does not terminate automatically. In the output above Enter has been pressed.
+
+
+**Direct Methods**
+
+This sample requires two actions: 
+- connecting the Azure IoT SDK device client to the hub, and 
+- triggering the direct method through the Azure Portal.
+
+First, run the sample:
+
+On the `az.sln` solution open on Visual Studio, 
+- Navigate on the Solution Explorer panel to `paho_iot_hub_methods_example` solution;
+- Make it the default startup project (right-click on `paho_iot_hub_methods_example` project, then click on `Set as StartUp Project`);
+- Build and run the project (`F5` on most installations).
+
+On the Azure Portal,
+- Go to your Azure IoT hub page.
+- Click on "IoT devices" under "Explorers".
+- From the list of devices, click on your device (created on step Z).
+- Click on "Direct Method".
+- On "Method Name", type "ping" (the sample expects the name "ping").
+- On "Payload", type '{ "somevalue": 1234 }' (the payload MUST be a valid Json).
+- Click on "Invoke Method".
+- See the reply from the sample on "Result" (bottom of the page).
+
+
+Back to the Visual Studio Command prompt, verify that the direct method has been received by the sample:
+
+```shell
+X509 Certificate PEM Store File = C:\azure-sdk-for-c\sdk\iot\hub\samples\src\device_cert_store.pem
+X509 Trusted PEM Store File = C:\azure-sdk-for-c\sdk\iot\hub\samples\src\BaltimoreCyberTrustRoot.crt.pem
+Device ID = paho-sample-device1
+IoT Hub Hostname = myiothub.azure-devices.net
+Posting connect semaphore for client paho-sample-device1 rc 0Subscribed to topics.
+Waiting for activity. [Press ENTER to abort]
+Topic: $iothub/methods/POST/ping/?$rid=1
+Direct Method arrived
+Status: 200     Payload:{"response": "pong"}
+Sent response
+```
+
+Note: the sample does not terminate automatically. In the output above Enter has been pressed.
+
+
+**Device Twin**
```

Comment:
Nit: The labels for the different types can be sub headers:
```suggestion
### Device Twin
```

Reply:
Sure, doing that for all the samples.